website/src/content/api-reference/commercetools-frontend-application-components.mdx

```diff
@@ -266,13 +266,13 @@ const Example = () => {
     <InfoModalPage
       title="Lorem ipsum"
       isOpen={true}
```

Comment:
I was wondering whether we should also use the modal state for the `isOpen` property.
Perhaps we can initialize the hook with `true` value and use it here so when we call the `closeModal` function, the modal would actually close.

```jsx
const Example = () => {
  const pageModalState = useModalState(true); // modal open by default

  return (
    <InfoModalPage
      title="Lorem ipsum"
      isOpen'{pageModalState.isModalOpen}
      ...
  );
}
```


What do you think?

Reply:
I added `isModalOpen` but I'm not sure about `useModalState(true)`. I mean only in terms of consistency, as we don't use it in other code snippets 🤔

Reply:
Ok, if we don't use it in other examples, maybe we should aim for consistency, yes.

It just caught my eye as the default value for the state is _false_ so when using `useModalState` with no params the `InfoModalPage` would not be visible until some code calls `pageModalState.openModal()`